dirsrvtests/tests/suites/plugins/accpol_test.py

```diff
@@ -1298,6 +1298,135 @@ def test_locact_modrdn(topology_st, accpol_local):
     account_status(topology_st, suffix, subtree, userid, 1, 0, "Enabled")
     del_users(topology_st, suffix, subtree, userid, nousrs)
 
+def test_acct_policy_consumer(topology_m1c1, request):
+    """Test the lastLoginHistory is updated on consumer without
+    referral error
+
+    :id: 53a9a2c7-6b10-41c9-b9f9-bde412d04acb
+    :setup: Supplier Instance, Consumer Instance
+    :steps:
+        1. Create a test entry on the supplier
+        2. Configure lastLoginTime on supplier and consumer
+        3. On supplier 3 binds of the test entry
+        4. On supplier check there is 3 lastLoginHistory values
+        5. On supplier check there is no error in error logs
+        6. On consumer 3 binds of the test entry
+        7. On consumer check there is 3 lastLoginHistory values
+        8. On consumer check there is no error in error logs
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+        8. Success
+    """
+
+    supplier = topology_m1c1.ms['supplier1']
+    consumer = topology_m1c1.cs['consumer1']
+    USER_PW = 'password'
+
+    # Add on the supplier a test user entry and wait it is on the consumer
+    users_supplier = UserAccounts(supplier, DEFAULT_SUFFIX, rdn=None)
+    user_supplier = users_supplier.create_test_user(uid=1000, gid=2000)
+    user_supplier.replace('userPassword', USER_PW)
+
+    users_consumer = UserAccounts(consumer, DEFAULT_SUFFIX, rdn=None)
+    for i in range(0, 10):
+        try:
+            user_consumer = users_consumer.get("test_user_1000")
+            break
+        except ldap.NO_SUCH_OBJECT:
+            time.sleep(1)
+
+    # Configure lastLoginTime on supplier and consumer
+    plugin = AccountPolicyPlugin(supplier)
+    try:
+        plugin.enable()
+        ACCPOL_DN = "cn={},{}".format(PLUGIN_ACCT_POLICY, DN_PLUGIN)
+        ACCP_CONF = "{},{}".format(DN_CONFIG, ACCPOL_DN)
+        plugin.set('nsslapd-pluginarg0', ACCP_CONF)
+        accp = AccountPolicyConfig(supplier, dn=ACCP_CONF)
+        accp.set('alwaysrecordlogin', 'yes')
+        accp.set('stateattrname', 'lastLoginTime')
+    except ldap.LDAPError as e:
+        log.error('Failed to enable Global Account Policy Plugin and Password policy attributes')
+        raise e
```

Comment:
Very minor, but I don't think we need these try/excepts - just let it raise the error when it happens.  Same for other code below.